sotodlib/core/resources.py

```diff
@@ -0,0 +1,62 @@
+import os
+from urllib.request import urlretrieve
+
+from ast import literal_eval
+
+
+RESOURCE_DEFAULTS = {
+    "de421.bsp": "ftp://ssd.jpl.nasa.gov/pub/eph/planets/bsp/de421.bsp",
+}
+
+
+def get_local_file(filename: str, cache: bool = True) -> str:
+    """
+    This function is responsible to check if a resource file exists either in a
+    predefined cache or in a user specified path. If not, it tries to download
+    from an ftp path.
```

Comment:
In docstring, explain the path search order.  Provide an example of a well-formed SOTODLIB_RESOURCES var.